lib/Runtime/Library/JavascriptProxy.cpp

```diff
@@ -228,20 +228,15 @@ namespace Js
         }
 
         PropertyId propertyId = getPropertyId();
-        CallInfo callInfo(CallFlags_Value, 3);
-        Var varArgs[3];
-        Js::Arguments arguments(callInfo, varArgs);
-        varArgs[0] = handlerObj;
-        varArgs[1] = targetObj;
-        varArgs[2] = GetName(requestContext, propertyId);
+        Var propertyName = GetName(requestContext, propertyId);
 
-        Assert(JavascriptString::Is(varArgs[2]) || JavascriptSymbol::Is(varArgs[2]));
+        Assert(JavascriptString::Is(propertyName) || JavascriptSymbol::Is(propertyName));
         //8. Let trapResultObj be the result of calling the[[Call]] internal method of trap with handler as the this value and a new List containing target and P.
         //9. ReturnIfAbrupt(trapResultObj).
         //10. If Type(trapResultObj) is neither Object nor Undefined, then throw a TypeError exception.
 
         Js::ImplicitCallFlags saveImplicitCallFlags = threadContext->GetImplicitCallFlags();
```

Comment:
>Js::ImplicitCallFlags saveImplicitCallFlags = threadContext->GetImplicitCallFlags(); [](start = 8, length = 84)

Isn't it better to have all such calls in this file use ExecuteImplicitCall instead? we don't want to allow this call if disableImplicitCall is set?

Reply:
Yes, that was going to be my next PR :)